### PR TITLE
Add runtime env manager

### PR DESCRIPTION
**Pull request by runzexia**
add runtime env manager proto, generate code, add server basic method

### CODE REVIEW

File: api/runtime_env.proto [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+	google.protobuf.StringValue labels = 3;
+	google.protobuf.StringValue runtime_env_url = 4;
+}
+
+message CreateRuntimeEnvResponse{
+	RuntimeEnv runtime_env = 1;
+}
+
+message DescribeRuntimeEnvsRequest{
+	repeated google.protobuf.StringValue runtime_env_ids = 1;
```
Comment (zheng1) [inline api/runtime_env.proto]: `runtime_env_ids`, you can replace it with `runtime_env_id`.

File: api/runtime_env.proto [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+	repeated google.protobuf.StringValue status = 2;
+	google.protobuf.StringValue search_word = 3;
+	google.protobuf.StringValue selector = 4;
+	google.protobuf.StringValue owner = 5;
```
Comment (zheng1) [inline api/runtime_env.proto]: `owner` can be repeated

File: api/runtime_env.proto [hunk 3]
```diff
@@ -0,0 +1,264 @@
...
+}
+
+message CreateRuntimeEnvCredentialRequset{
+	google.protobuf.StringValue name = 1;
+	google.protobuf.StringValue description = 2;
+	map <string, string> content = 3;
+}
+
+message CreateRuntimeEnvCredentialResponse{
+	RuntimeEnvCredential runtime_env_credential =1 ;
```
Comment (zheng1) [inline api/runtime_env.proto]: Add a space in front of 1.

File: api/runtime_env.proto [hunk 4]
```diff
@@ -0,0 +1,264 @@
...
+}
+
+message DescribeRuntimeEnvCredentialsRequset{
+	repeated google.protobuf.StringValue runtime_env_credential_ids = 1;
```
Comment (zheng1) [inline api/runtime_env.proto]: use `runtime_env_credential_id` replace `runtime_env_credential_ids `

File: api/runtime_env.proto [hunk 5]
```diff
@@ -0,0 +1,264 @@
...
+	repeated google.protobuf.StringValue status = 2;
+	google.protobuf.StringValue search_word = 3;
+	google.protobuf.StringValue owner = 4;
```
Comment (zheng1) [inline api/runtime_env.proto]: `owner` can be repeated

File: api/runtime_env.proto (hunk 1 above)
Comment (runzexia) [reply api/runtime_env.proto]: For repeated types of data, should we use the plural form of the word

Comment (runzexia) [reply api/runtime_env.proto]: In protobuf's offical doc, they use the plural form of words.

Comment (zheng1) [reply api/runtime_env.proto]: In a describe requests, `repeated` filter use single number. This approach references most cloud vendors.

File: pkg/manager/runtime_env/server.go [hunk 6]
```diff
@@ -0,0 +1,26 @@
+package runtime_env
```
Comment (chilianyi) [inline pkg/manager/runtime_env/server.go]: Need copyright

**Pull request merged**